web/src/components/layout/header/Navbar.vue

```diff
@@ -70,14 +72,15 @@ export default defineComponent({
     const authentication = useAuthentication();
     const { darkMode } = useDarkMode();
     const docsUrl = window.WOODPECKER_DOCS;
+    const apiUrl = `${config.rootURL ?? ''}/swagger/index.html`;
```

Comment:
See  #1992

I would merge mentioned pull first, and then refactor to use that config